### ISSUE TITLE
Patching instructions for versions < 1.9 regarding the MasterCard issue

### ISSUE THREAD

**Issue by korostii** [magento/devdocs#1174]
Feedback on page: http://devdocs.magento.com/guides/v2.1/release-notes/tech_bull_21x_MasterCard.html
The [SUPEE-8967 patch](https://magento.com/tech-resources/download#download2005) seems to be available for the older versions (below 1.9.0.0) as well: http://take.ms/L484F
However, the technical link seem to only mention the versions starting with 1.9.0.0
There still might be merchants running on versions older than 1.9, could you please clarify the versions supported by patch (I assume it's 1.5.0.0+ with SUPEE-2725 installed but can't be sure) ?

Providing direct links to the mentioned patches seems appropriate as well:
https://magento.com/tech-resources/download#download2005
https://magento.com/tech-resources/download#download268

**Comment by jfrontain** [comment]
Thanks, @korostii! I'll update that TB later today. Keep those comments coming -- I really appreciate your close reading of these docs.

**Comment by jfrontain** [comment]
@korostii, I updated this Technical Bulletin to correctly list M1.x CE information, and linked to the relevant patch. Closing now -- please re-open if you think of any other enhancement.

**Comment by korostii** [comment]
Puzzling enough, I can't see a "reopen" button on this issue. Funny enough, that seems to be a [known issue ](https://github.com/isaacs/github/issues/583) of the GitHub itself.

Anyway, there's one more little thing about the links, please compare the two:
The link on the devdocs page: https://magento.com/tech-resources/download
Direct to SUPEE-8967: https://magento.com/tech-resources/download#download2005

When you click the first one, you need to additionally switch to the "Release Archive" tab http://take.ms/azd5z , and then scroll to the middle of this really long page in order to find the patch you need (alternatively, you could search in-page for "8967" http://take.ms/Fjb01 - again, only after switching the tab)
From UX perspective, I believe the second link is better. And it's easy to generate, too: http://take.ms/PiUKk .

By the way, the "Access Magento 1.x patches" instruction doesn't make any sense to me, neither. 
Being a CE user, I don't see any "Download" links nowhere near My Account: http://take.ms/gRQWx

**Comment by jfrontain** [comment]
@korostii, thanks again! Following up...

**Comment by jcalcaben** [comment]
@jfrontain can you follow up on this issue?

**Comment by shrielenee** [comment]
Hello! We are archiving this ticket in an effort to create a quality, community-driven backlog which will allow us to allocate the required attention more easily.

It looks like your original issue was addressed, but there was another issue brought to our attention: "By the way, the "Access Magento 1.x patches" instruction doesn't make any sense to me, neither.
Being a CE user, I don't see any "Download" links nowhere near My Account: http://take.ms/gRQWx"
 
If you believe that the reported problem needs further review, please open a new issue regarding it so we may work it with efficiency. Thank you for your continued collaboration and assistance!
